src/js/components/Box/index.d.ts

```diff
@@ -38,7 +38,7 @@ export interface BoxProps {
   flex?: 'grow' | 'shrink' | boolean | { grow?: number; shrink?: number };
   fill?: FillType;
   focusIndicator?: boolean;
-  gap?: GapType;
+  gap?: GapType | object;
```

Comment:
```suggestion
  gap?: GapType | { row?: string; gap?: string };
```
can we be more specific?